src/lexer/token.ts

```diff
@@ -68,18 +68,14 @@ export const isToken = {
   WITH: testToken({ value: 'WITH', type: TokenType.RESERVED_COMMAND }),
 };
 
-/** Checks if token is a Reserved Command or Reserved Binary Command */
-export const isCommand = (token: Token): boolean =>
-  token.type === TokenType.RESERVED_COMMAND || token.type === TokenType.RESERVED_BINARY_COMMAND;
```

Comment:
Inlined this to AliasAs - the single place it's used. Also switched it to only check for `RESERVED_COMMAND`, which seems to be the only token type for which we really need to check in there.